functions-and-operators/bit-functions-and-operators.md

```diff
@@ -19,3 +19,12 @@ TiDB supports all of the [bit functions and operators](https://dev.mysql.com/doc
 | [^](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_bitwise-xor) | Bitwise XOR |
 | [<<](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_left-shift) | Left shift |
 | [>>](https://dev.mysql.com/doc/refman/8.0/en/bit-functions.html#operator_right-shift) | Right shift |
+
+# MySQL Compatibility
+
+There are some differences in how MySQL 8.0 and earlier versions of MySQL handle bit operations. TiDB aims to follow the MySQL 8.0 behaviour.
+
+## Known issues
+
+- [bitwise operations with binary args behavior changes by mysql 5.7 and 8.0 both](https://github.com/pingcap/tidb/issues/30637)
```

Comment:
Maybe change the title of the bug, or at least the reference here, since the title ends a bit weird with `... both`